src/System.Private.CoreLib/shared/System/TypeLoadException.cs

```diff
@@ -0,0 +1,76 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*=============================================================================
+**
+**
+**
+** Purpose: The exception class for type loading failures.
```

Comment:
Nit: We have been deleting useless comments like this when moving stuff to shared.